content/ja/docs/tasks/debug/debug-application/debug-pods.md

```diff
@@ -131,12 +132,12 @@ Podの`containerPort`がServiceの`targetPort`と一致することを確認し
 
 #### ネットワークトラフィックが転送されません
 
-詳しくは[Serviceのデバッグ](/ja/docs/tasks/debug-application-cluster/debug-service/)を参照してください。
+詳しくは[Serviceのデバッグ](/ja/docs/tasks/debug/debug-application/debug-service/)を参照してください。
 
 ## {{% heading "whatsnext" %}}
 
 上記のいずれの方法でも問題が解決しない場合は、以下の手順に従ってください。
-[Debugging Service document](/docs/tasks/debug-application-cluster/debug-service/)で、`Service` が実行されていること、`Endpoints`があること、`Pods`が実際にサービスを提供していること、DNS が機能していること、IPtablesルールがインストールされていること、kube-proxyが誤作動を起こしていないようなことを確認してください。
+[Serviceのデバッグに関するドキュメント](/ja/docs/tasks/debug/debug-application/debug-service/)で、`Service` が実行されていること、`Endpoints`があること、`Pods`が実際にサービスを提供していること、DNS が機能していること、IPtablesルールがインストールされていること、kube-proxyが誤作動を起こしていないようなことを確認してください。
 
-[トラブルシューティングドキュメント](/docs/tasks/debug-application-cluster/troubleshooting/)に詳細が記載されています。
+[トラブルシューティングドキュメント](/ja/docs/tasks/debug/)に詳細が記載されています。
```

Comment:
again, this link returns 404 not found
https://kubernetes.io/ja/docs/tasks/debug/

Reply:
A new page linked to ja/docs/task/debug is created with this PR. content/ja/docs/tasks/debug/_index.md

I can see it has generated on the preview.
https://deploy-preview-34162--kubernetes-io-main-staging.netlify.app/ja/docs/tasks/debug/

Reply:
ah sorry, so it's ok